pkgs/applications/emulators/dolphin-emu/default.nix

```diff
@@ -101,7 +101,8 @@ stdenv.mkDerivation rec {
     libevdev
     libXext
     libXrandr
-    mgba # Derivation doesn't support Darwin
+    # FIXME: Remove comment on next mgba version
+    #mgba # Derivation doesn't support Darwin
```

Comment:
Shouldn't this then be moved to a linux only block similar to 3 lines below for darwin?

Reply:
It already is in a Linux-only block.